src/internal/hooks/use-virtual/__tests__/use-virtual-offset.test.tsx

```diff
@@ -0,0 +1,133 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: why is this test file called "use-virtual-offset" and not "use-virtual" to match the util name?

Reply:
Because it only tests the offset right now and not the pre-existing functionality. I can rename it.